packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -98,6 +98,64 @@ enum DragAnchor {
 /// widget always displays [child].
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=QzA4c4QHZCY}
+/// 
+/// {@tool dartpad --template=stateless_widget_material}
+///
+/// The follwing example has the [Draggable] widget along with the [DragTarget] in a row demonstrating what happens when you drag the element to the target.
+///
+/// ```dart
+/// Widget build(BuildContext context) {
+///   var acceptedData = 0;
+///   return Row(
+///     mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+///     children : [
+///       Draggable(
+///         // Data is the value this Draggable stores.
+///         data : 10,
+///         child : Container(
+///           height : 100.0,
+///           width : 100.0,
+///           color : Colors.lightGreenAccent,
+///           child : Center(
+///             child : Text("I am the Draggable")
+///           )
+///         ),
+///         feedback : Container(
+///           height : 100.0,
+///           width : 100.0,
+///           color : Colors.deepOrange,
+///           child : Icon(Icons.directions_run)
+///         ),
+///         childWhenDragging : Container(
+///           height : 100.0,
+///           width : 100.0,
+///           color : Colors.pinkAccent,
+///           child : Center(
+///             child : Text("I am what's left behind.")
+///           )
+///         )
+///       ),
+///       DragTarget(
+///         builder : (BuildContext context) {
+///             Container(
+///               height : 100.0,
+///               width : 100.0,
+///               color : Colors.cyan,
+///               child : Center(
+///                 child : Text("Value is updated to: $acceptedData"),
+///               )
+///             )
+///          }
+///          onAccept : (int data){
+///            acceptedData += data;
+///          }
```

Comment:
This will need to be wrapped in a `setState`. Otherwise, the UI will not update with the value changes.
```dart
            onAccept: (int data) {
              setState(() {
                acceptedData += data;
              });
            },
```

Reply:
That's what I thought! However, regardless of it being wrapped in a stateless or statefull widget it doesn't seem to work.
Try this [DartPad ](https://dartpad.dev/b5c028995679069b7fd4b799f718bdef) for context.

Reply:
It's because you're using a StatelessWidget here. If you update it into a StatefulWidget it works:
```dart

class Game extends StatefulWidget {
  @override
  _GameState createState() => _GameState();
}

class _GameState extends State<Game> {
  int acceptedData = 0;

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text("Draggables"),
        backgroundColor: Colors.pink,
      ),
      body: Center(
        child: Row(
          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
          children: [
            Draggable<int>(
              data: 10,
              child: Container(
                height: 100.0,
                width: 100.0,
                color: Colors.pinkAccent,
                child: Center(
                  child: Text("Draggable"),
                ),
              ),
              feedback: Container(
                color: Colors.deepOrange,
                height: 100,
                width: 100,
                child: Icon(Icons.directions_run),
              ),
              childWhenDragging: Container(
                height: 100.0,
                width: 100.0,
                color: Colors.lightGreenAccent,
                child: Center(
                  child: Text("Child When Dragging"),
                ),
              ),
            ),
            DragTarget(
              builder: (
                BuildContext context,
                List<dynamic> accepted,
                List<dynamic> rejected,
              ) {
                return Container(
                  height: 100.0,
                  width: 100.0,
                  color: Colors.cyan,
                  child: Center(
                    child: Text("$acceptedData"),
                  ),
                );
              },
              onAccept: (int data) {
                setState(() {
                  acceptedData += data;
                });
              },
            )
          ],
        ),
      ),
    );
  }
}
```